packages/react-components/tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "allowSyntheticDefaultImports": true,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
-    "module": "esnext",
+    "module": "commonjs",
```

Comment:
I think we should be using `esnext` https://github.com/mui-org/material-ui/blob/b4d4a325c4fb4411276b2d5cd2b6f4779ea92b3a/tsconfig.json#L3. We probably need to change the webpack/karma config so that it works.

Reply:
Done!